src/data/icons.json

```diff
@@ -338,6 +338,13 @@
       "style": "ico_squareLarge",
       "category": "interface"
     },
+    {
+      "title": "Global Language",
```

Comment:
Why did you fill out all the things up here, but not in lines 564 and 565?

This also opened my eyes to a complex issue with the primaryBuild.js file. 
[Functionality is confusing #28](https://github.com/AlaskaAirlines/OrionIcons/issues/28)

Reply:
Honestly, I was following the previous pattern set forth by other items in the category. Would you like me to fill it out?

Reply:
That is what I was afraid of. There are real issues that were missed when all the functionality of this thing blew up. 

Yes, please fill out the rest and other clean up will happen with #28